### DISCUSSION TITLE
Question about "--no-ifs-removal" flag

### DISCUSSION THREAD

**Discussion by JianAtSeer** [vdemichev/DiaNN#576] [General]
I have noticed in the new version (1.8.1) there is this flag "--no-ifs-removal" being turned on by default from the gui?
Can you please provide a little more detail on what does this flag does?  It looks like it is turning off the 
interference removal step? I had thought this is an important feature for DIANN compared to other peptide-centric tool. Is there any guidance on in what scenario we should turn on/off interference removal? Thanks

**Comment by vdemichev** [comment]
Yes, for many scenarious not using interference removal is better.

**Reply by vdemichev** [reply]
Please don't change this setting unless you have independent evidence that it's better for your experimental setup.
